packages/prisma/schema.prisma

```diff
@@ -564,8 +568,10 @@ model BookingReference {
   externalCalendarId         String?
   deleted                    Boolean?
 
-  credential   Credential? @relation(fields: [credentialId], references: [id], onDelete: SetNull)
-  credentialId Int?
+  credential                       Credential?           @relation(fields: [credentialId], references: [id], onDelete: SetNull)
+  credentialId                     Int?
+  domainWideDelegation             DomainWideDelegation? @relation(fields: [domainWideDelegationCredentialId], references: [id], onDelete: Cascade)
```

Comment:
@Udit-takkar Just noticed that I have configured to delete it when DWD is deleted. I think that is wrong. We should configure it to have same behaviour as credential i.e. onDelete:SetNull.

It means that right now if we delete a DomainWideDelegation entry that would delete the BookingReference which could be used in some existing booking.